### PR TITLE
Credits v2 confirmation page

### PR DESCRIPTION
**Pull request by mauro-fanton**
### Description of Work carried through

Brief description of what/why/how.

### CODE REVIEW

File: app/controllers/returns/credits/ConfirmPackagingCreditController.scala [hunk 1]
```diff
@@ -16,37 +16,52 @@
...
-                                                  val controllerComponents: MessagesControllerComponents,
-                                                  view: ConfirmPackagingCreditView,
-  returnsNavigator: ReturnsJourneyNavigator
+  override val messagesApi: MessagesApi,
+  creditConnector: CalculateCreditsConnector,
+  identify: IdentifierAction,
+  getData: DataRetrievalAction,
+  requireData: DataRequiredAction,
+  val controllerComponents: MessagesControllerComponents,
+  view: ConfirmPackagingCreditView,
```
Comment (liampan) [inline app/controllers/returns/credits/ConfirmPackagingCreditController.scala:39]: can we rename this view, as there is 2

File: app/forms/returns/credits/ConvertedCreditsFormProvider.scala [hunk 2]
```diff
@@ -20,10 +20,15 @@ import javax.inject.Inject
 import forms.mappings.Mappings
 import play.api.data.Form
 import play.api.data.Forms.mapping
+import play.api.libs.json.{Json, OFormat}
 import uk.gov.voa.play.form.ConditionalMappings.{isEqual, mandatoryIf}
 
 case class ConvertedCreditsAnswer(yesNo: Boolean, weight: Option[Long])
```
Comment (liampan) [inline app/forms/returns/credits/ConvertedCreditsFormProvider.scala:26]: as this is now used in 2 places (form & cache) can this be moved to models? Check with carl I also suggested this on his pr he may have done it, and cause conflicts

File: app/forms/returns/credits/ExportedCreditsFormProvider.scala [hunk 3]
```diff
@@ -19,10 +19,16 @@ package forms.returns.credits
 import forms.mappings.Mappings
 import play.api.data.Form
 import play.api.data.Forms.mapping
+import play.api.libs.json.{Json, OFormat}
 import uk.gov.voa.play.form.ConditionalMappings.{isEqual, mandatoryIf}
 
 case class ExportedCreditsAnswer(yesNo: Boolean, weight: Option[Long])
```
Comment (liampan) [inline app/forms/returns/credits/ExportedCreditsFormProvider.scala:25]: same comment as ConvertedCreditsAnswer

File: test/controllers/returns/credits/ConfirmPackagingCreditControllerSpec.scala [hunk 4]
```diff
@@ -16,167 +16,125 @@
...
 
-  val validAnswer: BigDecimal = 1.25
-
-  private lazy val convertedPackagingCreditRoute =
-    controllers.returns.credits.routes.ConfirmPackagingCreditController.onPageLoad().url
-
-  private val exportCreditConnector = mock[ExportCreditsConnector]
-
-  private val view = mock[ConfirmPackagingCreditView]
+  private val sut = new ConfirmPackagingCreditController(
```
Comment (liampan) [inline test/controllers/returns/credits/ConfirmPackagingCreditControllerSpec.scala:48]: nice!

**Review by liampan — APPROVED** [review]
Looks good to me, 
couple of minor refactor suggestions :)

**Review by C-Thomp — APPROVED** [review]
looks good to me :)

File: app/controllers/returns/credits/ConfirmPackagingCreditController.scala (hunk 1 above)
Comment (mauro-fanton) [reply app/controllers/returns/credits/ConfirmPackagingCreditController.scala:39]: cannot see another ConfirmPackagingCreditView.scala.html. do you know where it is?

Comment (mauro-fanton) [reply app/controllers/returns/credits/ConfirmPackagingCreditController.scala]: ok, done

**Pull request merged**